.changeset/true-windows-scream.md

```diff
@@ -0,0 +1,5 @@
+---
+"@svenvw/fdm-core": minor
+---
+
+Add `b_lu_yield` as property of cultivation catalogue. It represents an average yield for the cultivation that can be used as default
```

Comment:
kg dry matter / ha or kg fresh yield per ha? or tons?

Reply:
Will be resolved in 572cd350cd8e2a61ad469f71212b7fa19d48f5b1